server/externs/mongodb.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @fileoverview Node.js mongodb externs.
+ */
+
+/** @const */
+var mongodb = {};
+
+/**
+ * @constructor
+ * @return {!mongodb.Cursor}
+ */
+mongodb.Cursor = function() {};
+
+/** @const */
+mongodb.MongoClient;
+
+/**
+ * @param {string} url
+ * @param {function()} callback
+ */
+mongodb.MongoClient.connect = function(url, callback) {};
+
+/**
+ * @constructor
+ * @return {!mongodb.Db}
+ */
+mongodb.Db = function() {};
+
+/**
+ * @param {string} dbName Name of database.
+ * @return {!Object}
+ */
+mongodb.Db.prototype.collection = function(dbName) {};
+
+/**
+ * @param {Object=} query
+ * @param {Object=} projection
+ * @return {!mongodb.Cursor}
+ */
+mongodb.Db.prototype.collection.find = function(query, projection) {};
+
+/**
+ * @param {!Object} document
+ * @param {Object=} writeConcern
+ */
+mongodb.Db.prototype.collection.insertOne = function(document, writeConcern) {};
+
+/**
+ * @param {!Object} query
+ * @param {!Object} update
+ * @param {boolean=} upsert
+ * @param {boolean=} multi
+ * @param {Object=} writeConcern
+ */
+mongodb.Db.prototype.collection.update = function(query, update, upsert, multi, writeConcern) {};
+
+/**
+ * @param {!Object} collection
+ * @param {!Object} query
+ * @param {function()=} callback
+ * @return {!Object} userInfo
+ */
+mongodb.Db.prototype.getOne = function(collection, query, callback) {};
```

Comment:
You confused me. Isn't it a wrapper function that genotet created? You wrote at database.getOne. I didn't find a method called getOne under the mongoDB doc or the mongo collection source code. I thought I would suggesting using a mongoDB function directly if that exists. What I found last time was that we had to write a wrapper.